migrations/2022-06-13-124806_post_report_name_length/down.sql

```diff
@@ -0,0 +1,2 @@
+-- not doing anything here because the previous type varchar(100) is wrong and breaks report functionality for long titles
+NULL;
```

Comment:
This is supposed to work as noop, but gives me an error. Empty file also doesnt work.

https://www.postgresql.org/docs/8.4/plpgsql-statements.html#PLPGSQL-STATEMENTS-NULL

```
Rolling back migration 2022-06-13-124806_post_report_name_length
Executing migration script /home/felix/workspace/lemmy/migrations/2022-06-13-124806_post_report_name_length/down.sql
Failed with: syntax error at or near "NULL"
```

Reply:
They need to be exact undos anyway, otherwise `diesel migration redo` will fail.